point_cloud_test/src/queries.rs

```diff
@@ -0,0 +1,39 @@
+// Some synthetic queries for synthetic data. These are just examples, more can be added.
+use crate::synthetic_data::SyntheticData;
+use crate::S2_LEVEL;
+use cgmath::{PerspectiveFov, Rad, Vector3};
+use point_viewer::iterator::PointLocation;
+use point_viewer::math::{Frustum, Obb, S2Point};
+
+pub fn get_abb_query(data: SyntheticData) -> PointLocation {
+    let abb = data.bbox();
+    PointLocation::Aabb(abb)
+}
+
+pub fn get_obb_query(data: SyntheticData) -> PointLocation {
+    let obb = Obb::new(
+        data.ecef_from_local().clone(),
```

Comment:
nit: maybe add a comment to explain how the obb is constructed (i.e. for the test purpose it is currently taken in the middle of the generated point cloud

Reply:
Done.